pkg/identityserver/application_registry.go

```diff
@@ -169,16 +169,18 @@ func (is *IdentityServer) listApplications(ctx context.Context, req *ttnpb.ListA
 		if err != nil {
 			return err
 		}
-		for i, app := range apps.Applications {
-			if rights.RequireApplication(ctx, app.ApplicationIdentifiers, ttnpb.RIGHT_APPLICATION_INFO) != nil {
-				apps.Applications[i] = app.PublicSafe()
-			}
-		}
 		return nil
 	})
 	if err != nil {
 		return nil, err
 	}
+
+	for i, app := range apps.Applications {
```

Comment:
How is this change related?

Reply:
Consistency. If we move the same logic out of the transaction for Gateways, we should also do it in the other places.